pkg/server/biz/accelerator/accelerate.go

```diff
@@ -48,6 +48,9 @@ func NewAccelerator(tenant, project string, wfr *v1alpha1.WorkflowRun) *Accelera
 // - '/root/.m2'  maven dependency path
 // - '/root/.gradle'  gradle dependency path
 // - '/root/.npm'  npm dependency path
+// - '/root/.sbt' sbt default configuration path
```

Comment:
Is `the configuration path` necessary, I think the sizes of configuration files will not be very large usually.

Reply:
I was considering configure repositories in path ```.sbt```. After some dig, I found it can be configured by sbt command line ```sbt -Dsbt.override.build.repos=true sbt -Dsbt.repository.config=<repositories-config-file-path>```.

Refer: https://stackoverflow.com/questions/3770125/how-do-i-get-sbt-to-use-a-local-maven-proxy-repository-nexus?rq=1

So, I will remove ```.sbt``` path